src/odemis/acq/stitching/_registrar.py

```diff
@@ -293,29 +308,31 @@ def _estimateMatch(self, imageA, imageB, shift=(0,0)):
 
         return covar / (stDev[0] * stDev[1])
 
-    def _get_shift(self, prev_tile, tile, shift=(0,0)):
+    def _get_shift(self, prev_tile, tile, shift=(0, 0)):
         """
         This is actually a wrapper of PyPhenom.GetShift that measures the shift on the overlap
         between the two tiles when the second is shifted by the shift value.
         shift (2 ints): shift between prev_tile and tile
         return (2 ints): guessed shift
         """
         (l1, t1, r1, b1), (l2, t2, r2, b2) = self._estimateROI(shift)
-        
-        a=prev_tile[t1:b1,l1:r1]
-        b=tile[t2:b2,l2:r2] 
-        
-        [x,y]=MeasureShift(b,a)
-        return x,y
+
+        a = prev_tile[t1:b1, l1:r1]
+        b = tile[t2:b2, l2:r2]
+
+        [x, y] = MeasureShift(b, a)
+        return x, y
 
     def _mean_shift(self, row, col, coords_array):
         """
         Returns the mean shift computed along the given row and column
         """
         val = [coords_array[row][i] for i in range(1, col) if coords_array[row][i] is not None] + \
-              [coords_array[i][col] for i in range(1, row) if coords_array[i][col] is not None]
+              [coords_array[i][col]
+                  for i in range(1, row) if coords_array[i][col] is not None]
         if not val:
-            raise ValueError("Not enough data to compute mean horizontal shift")
+            raise ValueError(
```

Comment:
Keep the line as a single line. It's more readable, and barely longer than 80 characters.